kim_commands.rst

```diff
@@ -1033,119 +1011,122 @@ property definition for more detailed information.
-structure.
-
-For example,
+The *kim\_property create* command takes as input a property instance ID and the
+property definition name, and creates an initial empty property instance data
+structure.  For example,
 
 .. parsed-literal::
 
    kim_property create 1 atomic-mass
    kim_property create 2 cohesive-energy-relation-cubic-crystal
 
-where "1" is a property instance ID, and "atomic-mass" is a property
-definition name to create the first property instance. "2" and
-"cohesive-energy-relation-cubic-crystal" are the property instance ID and
-property definition name to create the second property instance,
-respectively. See `properties page <https://openkim.org/properties>`_ for the
-list of each property definition that already exists in OpenKIM.
+creates an empty property instance of the "atomic-mass" property definition
+with instance ID 1 and an empty instance of the
+"cohesive-energy-relation-cubic-crystal" property with ID 2.  A list of
+published property definitions in OpenKIM can be found on the `properties page
+<https://openkim.org/properties>`_.
 
-One can also provide a file name in the current working directory or the
-pathname of the file to be opened. The file must conatin a valid property
-definition. (See
-`KIM Properties Framework <https://openkim.org/doc/schema/properties-framework/>`_
-for more detailed information.)
-
-For example,
+One can also provide the name of a file in the current working directory or the
+path of a file containing a valid property definition.  For example,
 
 .. parsed-literal::
 
    kim_property create 1 new-property.edn
 
 where "new-property.edn" refers to a file name containing a new property
-definition which does not exist in OpenKIM.
-
-In case of a wrong name or a wrong ID, or if the program can not find the
-file, running the script will fail with an error message indicating that the
-the requested name does not exist in OpenKIM nor a valid path-like object to
-a file can be found.
+definition that does not exist in OpenKIM.
 
-Calling the *kim\_property create* multiple times and using the same instance
-ID will fail with an error message indicating that the repeating instance ID
-is not allowed.
+If the *property\_id* given cannot be found in OpenKIM and no file of this name
+containing a valid property definition can be found, this command will produce
+an error with an appropriate message.  Calling *kim\_property create* with the
+same instance ID multiple times will also produce an error.
 
 **Destroy**
 
 .. parsed-literal::
 
    kim_property destroy instance_id
 
-*kim\_property destroy* command, deletes a previously created property
-instance ID.
-
-For example,
+The *kim\_property destroy* command deletes a previously created property
+instance ID.  For example,
 
 .. parsed-literal::
 
    kim_property destroy 2
 
+Note that if this command is called with an instance ID that does not exist, no
+error is raised.
+
 **Modify**
```

Comment:
Merge this with the above discussion of `modify` and remove any repeated content